features/shop/user/deleting_account.feature

```diff
@@ -0,0 +1,25 @@
+@user @ui
```

Comment:
Please move the `@ui` to each scenario (see [here](https://github.com/Sylius/Sylius/pull/4159)).